datasets/wino_bias/README.md

```diff
@@ -81,18 +79,19 @@ English
-    - predicate_lemma = The predicate lemma is mentioned for the rows for which we have semantic role information or word sense information. All other rows are marked with a "-".
-    - predicate_framenet_id = This is the PropBank frameset ID of the predicate in predicate_lemma.
-    - word_sense = This is the word sense of the word in Column tokens.
-    - speaker = This is the speaker or author name where available.
-    - ner_tags = These columns identifies the spans representing various named entities. For documents which do not have named entity annotation, each line is represented with an "*".
-    - verbal_predicates = There is one column each of predicate argument structure information for the predicate mentioned in predicate_lemma. If there are no predicates tagged in a sentence this is a single column with all rows marked with an "*".
+    - pos_tags = This is the Penn Treebank style part of speech. When parse information is missing, all part of speeches except the one for which there is some sense or proposition annotation   are marked with a XX tag. The verb is marked with just a VERB tag. [set to '-' currently]
+    - parse_bit = This is the bracketed structure broken before the first open parenthesis in the parse, and the word/part-of-speech leaf replaced with a *. The full parse can be created by substituting the asterix with the "([pos] [word])" string (or leaf) and concatenating the items in the rows of that column. When the parse information is missing, the first word of a sentence is tagged as "(TOP*" and the last word is tagged as "*)" and all intermediate words are tagged with a "*". [set to '-' currently]
+    - predicate_lemma = The predicate lemma is mentioned for the rows for which we have semantic role information or word sense information. All other rows are marked with a "-". [set to '-' currently]
+    - predicate_framenet_id = This is the PropBank frameset ID of the predicate in predicate_lemma. [set to '-' currently]
+    - word_sense = This is the word sense of the word in Column tokens. [set to '-' currently]
+    - speaker/author = This is the speaker or author name where available. [only "Speaker#1" currently]
+    - ner_tags = These columns identifies the spans representing various named entities. For documents which do not have named entity annotation, each line is represented with an "*". [set all to '*' now]
+    - verbal_predicates = There is one column each of predicate argument structure information for the predicate mentioned in predicate_lemma. If there are no predicates tagged in a sentence this is a single column with all rows marked with an "*". 
+    - coreference = Coreference chain information encoded in a parenthesis structure.
 
 ### Data Splits
 
-Single Split available
+- dev and test splits. For each split, there will be "pro_stereotype" and "anti_stereotype" splits. Each "pro_" and "anti_" splits will have "type1" and "type2" splits. 
```

Comment:
In the dataset script, only a train split is generated.
Where do these other splits come from ? Are they missing in the wino_bias dataset builder ?